BREAKING.md

```diff
@@ -46,6 +47,11 @@ It's not a breaking change, but worth noting: we are now also exposing optional
 Due to the dependency the Garbage Collection feature in the Runtime layer has on this policy, it must remain constant over time.
 So this has been codified in the type, switching from `number | undefined` to `FiveDaysMs | undefined` (with `type FiveDaysMs = 432000000`)
 
+### Static `FluidDataStoreRuntime.load` method is now deprecated
+
+Use `FluidDataStoreRuntime`'s constructor instead, and start providing the new `initializeEntrypoint` parameter
+to create the entrypoint / root object for the data store.
```

Comment:
Not sure how exactly can we improve this so people reading it can clearly understand what needs to be done, what that entrypoint / root object is...

Reply:
Looks self-explanatory to me - granted I have a lot of context.